packages/web/pages/dashboard.tsx

```diff
@@ -53,6 +54,8 @@ export const Grid = (): ReactElement => {
   const [gridLayouts, setGridLayouts] = useState(
     JSON.parse(JSON.stringify(originalLayouts)),
   );
+  useUser({ redirectTo: '/community/players', redirectIfNotFound: true });
```

Comment:
I think we should do this better than this. Currently the whole page loads and then we redirect. 
Instead we could show a loading screen until the user's logged in or connected.

Reply:
this line of code stayed from another PR, I can remove it here since this PR is about 404 page.

Reply:
I removed it @dan13ram